cylc/flow/scripts/install.py

```diff
@@ -171,14 +191,76 @@ def get_source_location(path: Optional[str]) -> Path:
     return search_install_source_dirs(expanded_path)
 
 
+async def scan(wf_name: str, ping: bool = True) -> None:
+    """Print any instances of wf_name that are already active."""
+    opts = Values({
+        'name': [f'{wf_name}/*'],
+        'states': {'running', 'paused', 'stopping'},
+        'source': False,
+        'ping': ping,  # get status of scanned workflows
+    })
+    active = [
+        item async for item in get_pipe(opts, None, scan_dir=None)
+    ]
+    if active:
+        n = len(active)
+        grammar = (
+            ["s", "are", "them"]
+            if n > 1 else
+            ["", "is", "it"]
+        )
+        print(
+            CylcLogFormatter.COLORS['WARNING'].format(
+                f'WARNING: {n} run%s of "{wf_name}"'
```

Comment:
What do you think of "NOTE" instead? Perhaps less aggressive
```suggestion
                f'NOTE: {n} run%s of "{wf_name}"'
```

Reply:
:+1: yep (I originally left WARNING off, then back tracked!)